base/ltcmd.dtx

```diff
@@ -3996,19 +4008,15 @@
           }
```

Comment:
Instead of

```
            \tl_set:Nx \ProcessedArgument
              { #2 = { \exp_not:o { \use:n #1 } } }
```

you could use

```
            \tl_set:Nx \ProcessedArgument
              { #2 = { \exp_not:n #1 } }
```

(after all, you know that `#1` is a single `n`-type by now)